apps/ledger-live-mobile/src/screens/Manager/AppsList/AppRow.tsx

```diff
@@ -78,15 +69,17 @@ export default memo(function AppRow({
     [app.bytes, deviceInfo.version, deviceModel, installedApp?.blocks],
   );
 
+  const { enabled: displayAppDeveloperName } = useFeature("myLedgerDisplayAppDeveloperName") || {};
```

Comment:
[The nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) seems like more relevant for this case.